02_show_class_table/show_class_table.php

```diff
@@ -0,0 +1,101 @@
+<?php
+require_once('../db.php');
+
+// DB接続
+try {
+   $pdo = new PDO($dsn, $user, $password);
+   $pdo->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+} catch (Exception $e) {
+   echo "<span class='error'>エラーがありました。</span><br>";
+   echo $e->getMessage();
+   exit();
+}
+
+$stmt_subject = $pdo->prepare('SELECT * FROM classes_subject');
+$stmt_teacher = $pdo->prepare('SELECT * FROM classes_teacher');
+$stmt_place = $pdo->prepare('SELECT * FROM classes_place');
+$stmt_subject->execute();
+$stmt_teacher->execute();
+$stmt_place->execute();
+
+$result_subject = $stmt_subject->fetchAll(PDO::FETCH_NUM);
+$result_teacher = $stmt_teacher->fetchAll(PDO::FETCH_NUM);
+$result_place = $stmt_place->fetchAll(PDO::FETCH_NUM); 

```

Comment:
[imo]
`FETCH_ASSOC`とするとカラム名をキーとして値がとれるので、こちらのほうがテーブル表に表示するときに可読性が高くなるかなーと思いました！(事情があってあえてこうしているのなら無視して下さい 🙏 )
配列のインデックスを指定するよりわかりやすくなるかなと思います！

```suggestion
$result_subject = $stmt_subject->fetchAll(PDO::FETCH_ASSOC);
$result_teacher = $stmt_teacher->fetchAll(PDO::FETCH_ASSOC);
$result_place = $stmt_place->fetchAll(PDO::FETCH_ASSOC); 
```

出力イメージ
```php
Array
(
    [0] => Array
        (
            [id] => 1
            [name] => Math
        )

    [1] => Array
        (
            [id] => 2
            [name] => English
        )
)

```